source/rafcon/gui/controllers/graphical_editor_gaphas.py

```diff
@@ -254,7 +254,13 @@ def _paste_clipboard(self, *event):
         """
         if react_to_event(self.view, self.view.editor, event):
             logger.debug("Paste")
-            gui_helper_state_machine.paste_into_selected_state(self.model)
+            cursor_position = None
+            if len(event) >= 3 and event[2]:
+                cursor_position = event[2]
+            else:
+                logger.warning("Got no paste position.")
```

Comment:
No, i don't know any situation in which this can occur, but since i am still not so familiar with the rafcon code, i decided to be better safe than sorry. So this situation may never occur at all.

Reply:
I would leave away the `else` part, as `paste_into_selected_state` can handle `cursor_position` being `None`

Reply:
okay.